contracts/modules/governance/MemberRoles.sol

```diff
@@ -33,14 +38,14 @@ contract MemberRoles is IMemberRoles, Governed, LegacyMasterAware {
   }
 
   MemberRoleDetails[] internal memberRoleData;
-  bool internal constructorCheck;
+  bool internal _unused2;
   uint public maxABCount;
   bool public launched;
   uint public launchedOn;
 
-  // This slot was previously used as a mapping (address => address payable)
-  mapping (address => address payable) public _unused2;
-  mapping(address => bool) public refundEligible;
+  mapping(address => address payable) public _unused3;
+  mapping(address => bool) public _unused4;
+  mapping(bytes32 => bool) public usedMessageHashes;
 
   modifier checkRoleAuthority(uint _memberRoleId) {
```

Comment:
looks like modifier can go away as well

Reply:
Why?

Reply:
discussed offline, this is a broader discussion and out of scope for this PR